### ISSUE TITLE
Urql code generator doesn't generate handler parameter for subscriptions

### ISSUE THREAD

**Issue by andyedinborough** [dotansimha/graphql-code-generator#2982]
I have a subscription similar to this:

```
subscription ItemUpdated {
    itemUpdated {
        id
        name
        email
    }
}
```

This will generate the following hook:

```
export function useItemUpdatedSubscription(
    options: Omit<Urql.UseSubscriptionArgs<ItemUpdatedSubscriptionVariables>, 'query'> = {}
) {
    return Urql.useSubscription<ItemUpdatedSubscription>({ query: ItemUpdatedDocument, ...options });
}
```

However, `useSubscription` accepts a second parameter to handle the event.  It appears the code that should be generated is this:

```
export function useItemUpdatedSubscription<TData = any>(
    options: Omit<Urql.UseSubscriptionArgs<ItemUpdatedSubscriptionVariables>, 'query'> = {},
    handler?: Urql.SubscriptionHandler<ItemUpdatedSubscription, TData>
) {
    return Urql.useSubscription<ItemUpdatedSubscription, TData>({ query: ItemUpdatedDocument, ...options }, handler);
}
```

This would allow the caller to pass a function like this:

```
    const [result] = useRequestUpdatedSubscription<ItemInfo[] | undefined>({}, (previous = [], data) => {
        return previous.map(x => (x.id === data.itemUpdated.id ? { ...x, ...data.itemUpdated } : x));
    });
```

See https://formidable.com/open-source/urql/docs/basics/#subscriptions for more detail.

**Comment by dotansimha** [comment]
Merged the PR, thanks @andyedinborough :) 

Alpha is available until a stable release: `1.8.4-alpha-d0d201d3.91`.

**Comment by dotansimha** [comment]
Fixed in 1.9.0
